File: Easy/2026-07-25-3536-Maximum-Product-of-Two-Digits/solution.js
var maxProduct = function(n) {
    const digits = String(n).split('').map(Number);
    let maxProduct = 0;

    for (let i = 0; i < digits.length; i++) {
        for (let j = 0; j < digits.length; j++) {
            maxProduct = Math.max(maxProduct, digits[i] * digits[j]);
        }
    }

    return maxProduct;
};

File: Easy/2026-07-25-3536-Maximum-Product-of-Two-Digits/solution.py
class Solution:
    def maxProduct(self, n: int) -> int:
        digits = [int(d) for d in str(n)]
        max_product = 0
        
        for i in range(len(digits)):
            for j in range(len(digits)):
                product = digits[i] * digits[j]
                if product > max_product:
                    max_product = product
        
        return max_product